client/src/pages/Home.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Fragment, useContext } from "react";
+import { Switch, Route, Redirect } from "react-router-dom";
+import UserContext from "../context/UserContext";
+
+import Navbar from "../components/Navbar";
+
+const Home = () => {
+  const { user } = useContext(UserContext);
+
+  // return user ? (
```

Comment:
I realized this logic might not work entirely:
- right now isn't the user always true, since user is initialized to an empty object 
- also, we'll need to handle a loading state - because it'll take time to get the user - this is probably something you can tackle when we actually make the API calls

Reply:
I currently have user initialized to null - but if that changes I'll revise this conditional!

I'll keep the loading state in mind!